templates/csharp/notification-http-timer-trigger-isolated/MessageHandler.cs.tpl

```diff
@@ -0,0 +1,34 @@
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.AspNetCore.Http;
```

Comment:
While referencing aspnetcore lib?
The azfunc sample references system.net and uses `HttpRequestData` instead of `HttpRequest`

Reply:
Ah, I see. ASP.NET Core integration works as well.
https://learn.microsoft.com/en-us/azure/azure-functions/dotnet-isolated-process-guide#aspnet-core-integration